build/azure-pipelines-rolling.yml

```diff
@@ -1,143 +1,130 @@
-- task: DotNetCoreCLI@2
-  displayName: 'build Microsoft.OData.ModelBuilder Unit test project'
-  inputs:
-    projects: '$(Build.SourcesDirectory)\test\Microsoft.OData.ModelBuilder.Tests\Microsoft.OData.ModelBuilder.Tests.csproj'
-    arguments: '--configuration $(BuildConfiguration) --no-incremental'
-
-- task: UseDotNet@2
-  displayName: Use .NET 3.x
-  inputs:
-    packageType: 'sdk'
-    version: '3.x'
-
-# Run the Unit test
-- task: DotNetCoreCLI@2
-  displayName: 'Unit Tests (Microsoft.OData.ModelBuilder.Tests.csproj) '
-  inputs:
-    command: test
-    projects: '$(Build.SourcesDirectory)\test\Microsoft.OData.ModelBuilder.Tests\Microsoft.OData.ModelBuilder.Tests.csproj'
-    arguments: '--configuration $(BuildConfiguration) --no-build'
-
-# because the assemblies are delay-signed, we need to disable
-# strong name validation so that the tests may run,
-# otherwise our assemblies will fail to load
-- task: Powershell@2
-  displayName: 'Skip strong name validation'
-  inputs:
-    targetType: 'inline'
-    script: |
-      & "$(snExe)" /Vr $(ProductBinPath)\$(mainDll)
-      & "$(snExe64)" /Vr $(ProductBinPath)\$(mainDll)
-      & "$(snExe)" /Vr $(testBinPath)\$(testDll)
-      & "$(snExe64)" /Vr $(testBinPath)\$(testDll)
-  enabled: false
-  
-# CredScan
-- task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@2
-  displayName: 'Run CredScan - Src'
-  inputs:
-    toolMajorVersion: 'V2'
-    scanFolder: '$(Build.SourcesDirectory)\src'
-    debugMode: false
-
-- task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@2
-  displayName: 'Run CredScan - Test'
-  inputs:
-    toolMajorVersion: 'V2'
-    scanFolder: '$(Build.SourcesDirectory)\test'
-    debugMode: false
-  
-# Microsoft.CodeAnalysis.FxCopAnalyzers is added into the project.
-# Typically, .NET Standard (.NET Core) project doesn't need the FxCop.
-
-# Needn't the AntiMalware@3 task?
-- task: AntiMalware@3
-  displayName: 'Run MpCmdRun.exe - ProductBinPath'
-  inputs:
-    FileDirPath: '$(ProductBinPath)'
-  enabled: false
-
-- task: BinSkim@3
-  displayName: 'Run BinSkim - Product Binaries'
-  inputs:
-    InputType: Basic
-    AnalyzeTarget: '$(ProductBinPath)\**\Microsoft.OData.ModelBuilder.dll'
-    AnalyzeSymPath: '$(ProductBinPath)'
-    AnalyzeVerbose: true
-    AnalyzeHashes: true
-    AnalyzeEnvironment: true
-
-- task: PublishSecurityAnalysisLogs@2
-  displayName: 'Publish Security Analysis Logs'
-  inputs:
-    ArtifactName: SecurityLogs
-
-- task: PostAnalysis@1
-  displayName: 'Post Analysis'
-  inputs:
-    BinSkim: true
-    CredScan: true
-    PoliCheck: true
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: MSSecurity-1ES-Build-Agents-Pool
+      image: MSSecurity-1ES-Windows-2019
+      os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
+    stages:
+    - stage: build
+      jobs:
+      - job: Release
+        steps:
```

Comment:
I see tasks that are repeated between nightly and rolling yml files. Those can be moved to a separate template that can be linked to both files

Reply:
Given that this is the legacy version (release-1.x) and we don't expect changes to the pipeline, I don't think it's necessary in this case.